pkg/virt-launcher/virtwrap/api/converter_test.go

```diff
@@ -250,6 +250,7 @@ var _ = Describe("Converter", func() {
     <baseBoard></baseBoard>
   </sysinfo>
   <devices>
+    <emulator>/usr/bin/qemu-system-x86_64</emulator>
```

Comment:
Instead of this it sounded as if we could just default to the correct architecture in the domxml.